README

```diff
@@ -80,6 +80,10 @@ To install pip and Quart
  - python3 get-pip.py
  - pip3 install --user quart
 
+The gRPC system tests (tests/system_tests_grpc.py) use grpcio and protobuf modules.
+To install them use:
+  - pip3 install --user grpcio protobuf
```

Comment:
maybe add a script (or just command to this readme) that can be used to regenerate the autogenerated .py files

Reply:
Adding instructions to the README, please have a look.

Reply:
looks good; that does not add the copyright header, though, if I understand correctly

ok, I see you excluded the generated files from rat check